bin/format.sh

```diff
@@ -0,0 +1 @@
+yapf -ri confero
```

Comment:
I like to put little commands like this into scripts, because remembering things is hard, and being able to `ls bin` to see all the dev commands is handy